workspaceitem-data-license.md

```diff
@@ -0,0 +1,171 @@
+# WorkspaceItem data of license sectionType
+[Back to the definition of the workspaceitems endpoint](workspaceitems.md)
+
+The section data represent the data about the granted deposit license
+
+```json
+{
+  	"url": "https://dspace7.4science.it/dspace-spring-rest/api/core/bitstreams/004a297e-fd06-4662-ae51-73e4b7c165c8/content",
+    "acceptanceDate": "2017-11-20T10:32:42Z"
+}
+```
+
+* url (**READ-ONLY**): the URL where the accepted license is visible
+* acceptanceDate: the timestamp of the acceptance as recorded in the dcterms:dataAccepted metadata of the license bitstream. For license accepted before than DSpace7 the acceptanceDate will be *null* 
+
+## Patch operations
+The PATCH method expects a JSON body according to the [JSON Patch specification RFC6902](https://tools.ietf.org/html/rfc6902)
+
+Each successful Patch operation will return a HTTP 200 CODE with the new workspaceitem as body 
+
+### Add
+To accept a license the timestamp of the acceptance the client must send a JSON Patch ADD operation as follow
+
+`curl --data '{[ { "op": "add", "path": "/sections/<:name-of-the-form>/acceptanceDate", "value": "YYYY-MM-DDTHH:MM:SSZ"}]}' -X PATCH ${dspace7-url}/api/submission/workspaceitems/<:id>`
+
+for example, starting with the following document  
+```json
+{
+	id: 1,
+	type: "workspaceitem",
+	sections:
+	{
+		"traditional-page1":
+			{
+			  "dc.title" : [{value: "Sample Submission Item", language: "en"}],
+			  "dc.contributor.author" : [
+			  	 		{value: "Bollini, Andrea", authority: "rp00001", confidence: 600}
+			  ]
+			},
+		"license":
+			{
+			  	"url": null,
+			    "acceptanceDate": null
+			}
+		...
+	},
+	...
+}	
+```
+
+the following request 
+`curl --data '{[ { "op": "add", "path": "/sections/license/acceptanceDate", "value": "2017-11-20T10:32:42Z"}]}' -X PATCH ${dspace7-url}/api/submission/workspaceitems/1`
```

Comment:
Maybe we should change this to `{ "op": "add", "path": "/sections/license/accepted", "value": true}` and let the backend determine the acceptance date.

Reply:
This will be a major violation / customization of the json patch spec. The client should expect to have true as value of the` /sections/license/acceptanceDate` attribute after this call but we want to expose the timestamp

Reply:
after further discussion with @lap82 he has the same concerns than you about allowing the client to set the acceptanceDate... moreover the dspace < 7 implementation will force the date to the server time.
Do you think is it better to use the accepted path to notify the acceptance and reject of the license and keep the acceptanceDate path as "readonly"?

Reply:
I agree to leave acceptanceDate path as "readonly" and let the backend to determine the acceptance date. I suggest to use the path "/sections/license/grant".

Reply:
That's a good idea @lap82 Thanks